drivers/pwm/Kconfig.max2221x

```diff
@@ -0,0 +1,20 @@
+# Copyright (c) 2025 Analog Devices Inc.
+# SPDX-License-Identifier: Apache-2.0
+
+config PWM_MAX2221X
+	bool "Analog Devices MAX2221X multi-function device driver"
+	default y
+	depends on DT_HAS_ADI_MAX2221X_PWM_ENABLED
+	depends on MFD
+	help
+	  Enable driver for MAX2221x PWM
+
+#if PWM_MAX2221X
+
+config PWM_MAX2221X_INIT_PRIORITY
+	int "Init priority"
+	default 80
+	help
+	  Analog Devices MAX22216/MAX22217 PWM driver initialization priority.
+
+#endif #PWM_MAX2221X
```

Comment:
Is there a reason not to use `PWM_INIT_PRIORITY`?

If you do need a driver-specific init priority, you need to uncomment the if/endif statements and convert `config PWM_MAX2221X` to a menuconfig